src/common/disk.rs

```diff
@@ -376,3 +432,111 @@ impl fmt::Display for DiskKind {
         })
     }
 }
+
+/// Used to determine what you want to refresh specifically on the [`Disk`] type.
+///
+/// ⚠️ Just like all other refresh types, ruling out a refresh doesn't assure you that
+/// the information won't be retrieved if the information is accessible without needing
+/// extra computation.
+///
+/// ```no_run
+/// use sysinfo::{Disks, DiskRefreshKind};
+///
+/// let mut disks = Disks::new_with_refreshed_list_specifics(DiskRefreshKind::new());
+///
+/// for disk in disks.list() {
+///     assert_eq!(disk.total_space(), 0);
+/// }
+/// ```
+#[derive(Clone, Copy, Debug)]
+pub struct DiskRefreshKind {
+    kind: bool,
+    total_space: bool,
+    available_space: bool,
+    is_removable: bool,
+    is_read_only: bool,
+    usage: bool,
```

Comment:
So in here, I think there should be only three fields: the `kind` update, the space/removable/read_only one, and finally the `usage` (which should be renamed `io_usage`).

No need to be this specific I think.

Reply:
Sure. Any thoughts on what to name the field for space+removable+read_only? `details` or something?

Reply:
Alright, we'll go with that :)